inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/workload/DynamicWorkloadManagementPage.html

```diff
@@ -102,8 +102,8 @@ <h1>
                       </div>
                     </div>
                     <div class="card-footer">
-                      <input class="btn btn-primary" wicket:id="Search" type="submit" /> 
-                      <input class="btn btn-primary" wicket:id="Reset" type="submit" />
+                      <input class="btn btn-primary" wicket:id="Search" type="submit" value="Search" />
```

Comment:
Better fetch the string from the properties file. Also IMHO better use `<button>` we do elsewhere in the UI. Best check the HTML for other forms we use.